src/controller.js

```diff
@@ -121,13 +120,14 @@ class Controller {
   }
 
   async typeHackOn() {
-    console.log(this.hackOn);
     for (var i = 0; i < this.hackOn.length; i++) {
-      await this.typeCharacter(this.hackOn[i], null, 0);
-      if(i === 74) {
```

Comment:
This line `i === 74` is check if the index of the `for` loop is at index 74. I'm assuming you realized all the lines were 74 characters, in case `i % 74 === 0` would have worked for all line breaks. However, I changed the check to `this.hackOn[i] === '\n'`. This checks if the character it sees is a new line and acts appropriately, similar to how we handle this in `handleCharacter` by detecting new lines and calling `this.addNewLine`